src/AWS.Deploy.CLI/Commands/DeployCommand.cs

```diff
@@ -130,6 +130,19 @@ public async Task ExecuteAsync(string applicationName, string deploymentProjectP
 
             await CreateDeploymentBundle(orchestrator, selectedRecommendation, cloudApplication);
 
+            if (!string.IsNullOrEmpty(saveSettingsFilePath))
+            {
+                var errorMessage = await _deploymentSettingsHandler.SaveSettings(saveSettingsFilePath, selectedRecommendation, cloudApplication, _session);

```

Comment:
error messages should not be returned this way. We should use exceptions to present invalid state.

Reply:
Changed ` _deploymentSettingsHandler.SaveSettings(...)` to throw an expected exception